<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageSavedSettings</class>
 <widget class="QDialog" name="ManageSavedSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitor Settings</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-desktop-display">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Saved settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListWidget" name="allConfigs"/>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="renamePushButton">
            <property name="text">
             <string>Rename</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deletePushButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="applyPushButton">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QTextEdit" name="selectedSettingsTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
